protocol/messenger.go

```diff
@@ -3231,6 +3231,39 @@ func (r *ReceivedMessageState) addNewActivityCenterNotification(publicKey ecdsa.
 	return nil
 }
 
+func (r *ReceivedMessageState) addNewActivityCenterNotificationV2(publicKey ecdsa.PublicKey, m *Messenger, message *common.Message, responseTo *common.Message, albumId string) error {
```

Comment:
@ibrkhalil, could you explain the idea behind this V2 naming? Why can't we change the original function and have only the new behavior?

Reply:
You're totally right, I am going to extend the original function to use the correct var instead of useless overloading.

Reply:
You're totally right, I am going to extend the original function to use the correct var instead of useless overloading.

Reply:
Used a different way to do the same thing, Kindly take a look.